### DISCUSSION TITLE
Unconfirmed balance

### DISCUSSION THREAD

**Discussion by Prodigygr** [Ride-The-Lightning/RTL#1268] [Troubleshooting Questions] [answered]
I have unconfirmed sats. In RTL i have the lease option. I will take ma sats if i use it?
I have Umbrel and RTL version 0.13.2-beta on Rasp Pi4
[I have see this thread, but i do not know if applies to me.](https://github.com/Ride-The-Lightning/RTL/discussions/1190)
Thank you

UTXO Information
Transaction ID
38dfb46a0247f5d0e42f51db42f6aec7fc34d63cd4a0c73e922a67f7106c84c3
Label
0:sweep
Output Index
0
Amount (Sats)
3,086,250
Confirmations
0
Address Type
Taproot Pubkey Hash
Address

PK Script

**Comment by saubyk** [comment]
Hi @Prodigygr lease option is not applicable here
You can try to CPFP your unconfirmed transaction via command line. But ensure that you pay high enough fee so that you can get confirmed quickly

**Comment by Prodigygr** [comment]
Ok. I have [this transaction](https://mempool.space/tx/38dfb46a0247f5d0e42f51db42f6aec7fc34d63cd4a0c73e922a67f7106c84c3) with my unconfirmed balance, witch is 0.03086250BTC.
But the transaction above, replaced with [this transaction](https://mempool.space/tx/668b33eb4adb4f596371dd0551e9f5513fc03de847b822b6d1e937aee80d2f7c) and with 0.00002286BTC.
On my Umbrel and Ride The Lightning app, i see 0.030863btc unconfirmed.
CPFP what?
Thank you!

**Reply by ziggie1984** [reply, marked as answer]
In this case, you need to restart your LND node or wait until the RBF is confirmed, otherwise the big utxo of `f8713d976e6ed39b1aafee73e8b2fb462daf6fa6124b4d17ba1f360519165164:1` will be not available for you.
